db/migrate/20190321223558_remove_encrypted_authentication_from_publishers.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveEncryptedAuthenticationFromPublishers < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :publishers, :encrypted_authentication_token, :string
```

Comment:
What's the migration plan for moving this data over?

Reply:
None. These tokens are only used once. Affected users will have to re-auth once more